src/components/PriceCard.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <v-card>
+    <v-card-item>
+      <template v-slot:prepend>
+        <v-avatar v-if="price.product" :image="price.product.image_url || defaultAvatar"></v-avatar>
+        <v-avatar v-if="!price.product" :image="defaultAvatar"></v-avatar>
+      </template>
+
+      <v-card-title v-if="price.product">{{ price.product.product_name }}</v-card-title>
+      <v-card-title v-if="!price.product">{{ price.product_code }}</v-card-title>
+
+      <v-card-subtitle v-if="price.product">{{ price.product.product_quantity }} · {{ price.product.brands }}</v-card-subtitle>
```

Comment:
the field `product.brands` is currently being added in the backend : https://github.com/openfoodfacts/open-prices/pull/93